testing/bdd/applied-processing.py

```diff
@@ -0,0 +1,18 @@
+from pytest_bdd import when, scenarios
```

Comment:
I think it might matter that the filename of this begins `test_` - it looks like this file is a replacement for `test-trace.py` that has been removed. I believe that pytest.bdd looks for things prefixed `test`, though that might be my faulty memory.